features/steps/steps.ls

```diff
@@ -24,12 +24,9 @@ module.exports = ->
 
 
   @Given /^an instance of this service$/, (done) ->
-    port-reservation
-      ..get-port N (@service-port) ~>
-        @exocom.register-service name: 'tweets', port: @service-port
-        @process = new ExoService service-name: 'tweets', exocom-port: @exocom.pull-socket-port, exorelay-port: @service-port
-          ..listen!
-          ..on 'online', -> done!
+    @process = new ExoService exocom-host: 'localhost', service-name: 'tweets', exocom-port: @exocom-port
+      ..listen!
+      ..on 'online', -> wait 10, done # Wait for ExoCom to register the service
```

Comment:
This creates a potentially flaky test. Please add a TODO here to use `MockExoCom.wait-until-service` once that is shipped. I'm currently working on this.